pages/dkp/kommander/1.4/release-notes/1.4.1/index.md

```diff
@@ -0,0 +1,58 @@
+enterprise: false
+---
+
+<!-- markdownlint-disable MD034 -->
+
+**D2iQ&reg; Kommander&reg; version 1.4.1 was released on June 9, 2021.**
+
+[button color="purple" href="https://support.d2iq.com/s/entitlement-based-product-downloads"]Download Konvoy[/button]
+
+To get started with Kommander, [download](/dkp/konvoy/latest/download/) and [install](/dkp/konvoy/latest/install/) the latest version of Konvoy.
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>
+
+# Release Summary
+Kommander provides a command center for all your cloud native management needs in public Information as a Service (IaaS), on-premises, and edge environments. Kommander provides a multi-tenant experience to create, secure, and configure Kubernetes clusters and cloud native workloads. Kommander enables teams to unlock federated cost management across multiple clusters, whether they are a new Konvoy cluster or an existing 3rd party/DIY distribution installation.
+
+# Known issues
+
+In Kommander 1.3.2 and earlier versions, the Kubecost cost analyzer requests a PVC of size 0.2Gi by default. This can cause issues with provisioners requiring a minimum storage size of 1Gi. Upgrade to version 1.4.1 to resolve this issue. Refer [here](https://github.com/kubecost/docs/blob/master/storage.md) for more information. 
+
+# Breaking changes
+
+## Docker hub rate limiting
+Docker Hub announced an update to their image pull policies in August, 2020. The change results in the need to change cluster configurations to accommodate new account structures that enable image pull rate limiting.
+
+Rate limiting happens on a per-pull basis regardless of whether the pulled image is owned by a paid user. This means D2iQ, as owner of most images used in Konvoy, does not have any influence as to whether your current address is rate-limited or not. Konvoy does not have a strict dependency on Docker Hub accounts or plans.
+
+For more information on addressing this limit, see [Docker hub rate limits](../operations/manage-docker-hub-rate-limits).
+
+#### Component Versions
+- Addon: 1.4.1-4
+- Chart: 0.35.1
+- kommander-federation (yakcl): 0.18.0
+- kommander-licensing (yakcl): 0.18.0
+- UI: 6.99.2
+- kommander-karma: 0.3.19
+- kubeaddons-catalog: 0.1.16
+- kommander-thanos: 0.1.22
+- kubecost: 0.13.0
+- kubefed: 0.7.0
+- grafana: 6.6.0
+- karma: 0.70
+- thanos: 0.17.1
+- cost-analyzer: 1.81.1
+
+## Fixes and Improvements 
+
+- Update federated Kubecost to v0.13.0. This includes an upgrade of Prometheus to v2.24.0.
+- Update federated kubeaddons to v0.27.0.
+- Increase Kubecost default cost-analyzer persistent volume size to 32Gi. Upgrades may require manual intervention if your provisioner does not support volume expansion.
```

Comment:
```suggestion
- Increase the default Kubecost cost-analyzer PVC storage size from 0.2Gi to 32Gi to resolve deployment issues that occurred with provisioners that require a minimum size of 1Gi. See [Known issues](#known-issues) above if upgrading from a previous version. (COPS-6937)
```
just to match the note we wrote in https://github.com/mesosphere/dcos-docs-site/pull/3614/files#diff-0aa1fd4ce97a8f0c41ff7265803f380637352143ef29bdee8d1a32670592e797R29